distribution/feature-packs/server-feature-pack/src/main/resources/modules/system/layers/keycloak/org/keycloak/keycloak-model-legacy-private/main/module.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2016 Red Hat, Inc. and/or its affiliates
```

Comment:
Nitpick: year - same for all other new `module.xml` files found here.

Reply:
Fixed in this PR commit 7e21088b3875f92493fc215a623b6c417ba82379